src/_scss/pages/agencyV2/overview/_overview.scss

```diff
@@ -46,7 +46,10 @@
             
         }
         .agency-overview__image {
+            max-width: rem(90);
+            max-height: rem(90);
             height: 100%;
+            object-fit: contain;
```

Comment:
This is not supported by IE https://caniuse.com/object-fit

Reply:
No, I'm getting a bunch of API errors there...

Reply:
Yeah I think worst case some of the logos will be slightly stretched in IE, but most of them are in the realm of 70 x 70px (with 10px padding either side)